### DISCUSSION TITLE
[참여자들이 해야 할 일 & 회의록 ] 과거자료

### DISCUSSION THREAD

**Discussion by HangryDev** [Pseudo-Lab/Anxious-Lookout#17] [Announcements]
# 초조한 전망대 목표! 

### 우리의 자세는

- 서로 경청해주고, 즐거운 분위기를 만듭니다 
- 가짜연구소에 낄 수 있도록 적극 도와드리고, 자기들의 플젝을 진행하게끔 독려해여
- 이걸 계속해서 서로 리마인드 해 주자구여! 
 
## 루틴

- 서클 소개 & 온보딩 미션 (3분)
- 돌아가며 자기소개 (새 멤버 있으면)
- 가져온 뉴스 자유 토론
    - 뉴스는 다 같이 읽고, 수다는 사람이 많으면 소그룹으로 나누어 진행. 
    - (온라인) 멤버들 - 온보딩 미션 챙겨주기
    - (오프라인 있는 경우) 네트워킹 관련 일들 하자
- 끝나고
    - 로깅, 시작 전 홍보, 게스트 연락 받고 안내

### 뉴스 가져와서 로깅 

1. 각자 관심 있는 블로그들, 뉴스 사이트들 찾아서 파고들기.
(어디서 시작할 지 모르겠다면, 여기에서 관심 가는 주제의 채널을 찾아볼 수 있습니다. https://github.com/Pseudo-Lab/impatient-observatory/discussions/4 )
2. 금요일 5시까지 찾은 내용을 카톡방 & https://github.com/Pseudo-Lab/impatient-observatory/discussions/25 에 업로드! **링크 + 왜 중요한지** 포함, 한 단락으로 포함 

### 온보딩 미션 - 각자 진행
- https://github.com/Pseudo-Lab/impatient-observatory/discussions/11 를 확인하세요! 

### 로깅 

- [x]  오늘 합니다! 글 올리기 🍉홍규님
- [ ]  **뉴스레터** 형태로 만들기 
- 배포! 
- [ ]  #자유게시판 업로드 🐨
- [x] 모임 후기! 깃헙에 업로드 🐸 호현님

### 오프라인 행사 준비 & 진행

- [x]  공간 대여 & 준비 & 끝나고 놀 장소 섭외 👯‍♂️ ->

좀 더 큰 행사 준비시 이 리스트 참고 : https://docs.google.com/spreadsheets/d/1IpRfQBqzKRrwD0lkBPi1y8kaU8iB501LpOe1ulIpYfs/edit?gid=0#gid=0

### ⭐ 책임 이양!

새로 오시는 분들이 참여할 수 있도록, 맡은 일을 하나씩 넘겨 드립니다! 잘 해오는지 체크를 해 주시면 되겠습니다.  
- 혹시 독립하거나 나가고 자리가 비면, 다시 채워주시면 됨

---
# 1회성 할일들 
- [ ] 뉴스레터 만들기
- [ ] 온보딩 프로세스 누수 탐지- 들어오신 분들이 모임에 나오기, 나오고 활발하게 활동하기까지 있음. (백프로 참여하게 만들 수는 물론 없음.) 설명이 불충분해서 등등이 있을 것. 참여자 입장을 물어보고, 프로세스 중 개선할 수 있는 부분을 찾아보자.

**Comment by HangryDev** [comment]
# 6/22 첫 번째 회의! 

## 오늘 아젠다

1. 자기소개, 인사 
2. 기획, 공간 소개
    - 프론트원
    - 온라인  https://github.com/Pseudo-Lab/impatient-observatory/discussions
3. 매주 할 일, 역할 설명
4. 약식 리허설
    1. 처음 오면, 오프닝 설명

# 매 주 할 일!

## 루틴

- 서클 소개 & 온보딩 미션 (3분)
- 돌아가며 자기소개 (새 멤버 있으면)
- 뉴스 읽고 자유 토론
    - (온라인) 멤버들 - 온보딩 미션 챙겨주기
    - (오프라인 있는 경우) 네트워킹 관련 일들 하자
- 끝나고
    - 로깅, 시작 전 홍보, 게스트 연락 받고 안내

## 마스터들 할 일

<aside>
💡 첫 4주는 전부 참석! 이후 일종의 스케쥴 근무로 전환함.

</aside>

### ⭐ 온보딩 미션 관련, 질문 받아주기

### ⭐⭐⭐ 관심, 경청과 대화

- 이야기를 이끌어내는거라고 생각해요
    - 말 많이 시키고, 관심을 가져주자
- 제가 사람에 관심이 없는 편어서 제일 도움이 많이 필요합니다

### ⭐ 책임 이양!

중요. 뒤에 오시는 분들에게 하나씩 일을 넘깁니다. 

넘긴 뒤 잘 해오는지 체크를 해 주시면 되겠습니다.  

- 혹시 독립하거나 나가고 자리가 비면, 다시 채워주시면 됨

### ⭐⭐⭐⭐⭐그랜드마스터 쪼아줘야 합니다

제가 자꾸 간과, 까먹습니다

### 외교

외부의 전문가들, 같이 이야기 나누고 싶은 사람들을 초빙합니다 

### 약간의 발제 (처음에만)

- 이건 오늘 보여드립니다
- 어떤 분위기인지 잡아가는거

---

### 로깅

- [x]  오늘 합니다! 글 올리기 🍉
- [x]  연락 왔는지 체크 디스코드 🐨  /깃헙 🐸 → 게스트 연락받아 채널로 안내하기
- [x]  깃헙 디스커션 댓글에 멤버들이 찾아온 자료 정리 🐨 
- [x]  디스코드: ‘찌라시’ 형태로 만들어서 #자유게시판 업로드 🐨 
    - 이번 주 멤버들이 얘기한 내용 - 클로바로 요약
    - 아래에 링크 남김! “온보딩이 필요하시다면 이 링크를 확인해주세요!”
- [x] 모임 후기! 분위기, 참석 멤버 등 네이버 리뷰처럼 올립시다. 깃헙과 #자유게시판 업로드 🐸

### 오프라인 행사 준비 & 진행

- [x]  프론트원 공간 대여 & 공간 준비 (ex 책상 밀어놓고 에어컨 트는거) 👯‍♂️
- [x]  (오프라인 있는 경우) 끝나고 놀 장소 섭외 👯‍♂️

## 참여자 할 일

### 뉴스 가져오기

- 관심 있는 블로그들, 뉴스 사이트들 찾아서 파고들기.

하나씩 영역을 정하자

🍉 hackernews
🐨 
🐸 
🐃 ailinksantools
👯‍♂️ 


### 디스코드 / 단톡방 활성화
- 단톡방 주소: https://open.kakao.com/o/giNkBLyg

- [ ]  뉴스 올리면 거기서 댓글 달아주기

### 로그 남기기

- [ ]  참여 전 찾은 내용을 **discussion/아카이브**에 업로드 (형식 자유)

### 온보딩 미션 - 각자 진행

### 미니 플젝 (멤버들이 관심 있는 프로젝트을 피워낸 이후 )

- [ ]  ex) perplexity, gamma, copilot 등 강의 찾고 같이 듣기

# One-time Todo → 다음 주말까지

- [x]  마스터 분들과 진행 합을 맞춰보자 🐃 
- [x]  컨트리뷰터 추가 🐃 
- [x] 이 문서 보완, 깃허브에 업로드  🐸  🐨 
- [x] 온보딩 미션 추가와 보완  🐸  🐨 
- [x]  인원 예측 - 오실 건지 RSVP 하기 🍉 
- [x]  채널 정해야 함. 오픈카톡방 만들기
- [x]  계급 기준 - 정하기


---

### 슈도랩 블로그 작성 방법

> 빌더가 기획, 중간, 마무리 블로그를 작성함으로써 프로젝트/빌더/러너의 홍보 효과가 있고, Building in Public을 통해 성장할 수 있으며, 러너의 자료들과의 상호적인 링크로 유입을 늘리며 공유의 가치를 극대화할 수 있습니다. 가짜연구소 깃헙/블로그에 기록을 남기며 오픈소스에 “기여”를 직접적으로 시작하실 수 있습니다! 이 두가지 활동은 필수로 진행됩니다.
> 


### 1) Fork

https://github.com/pseudolab/pseudolab.github.io

<img width="1253" alt="image" src="https://github.com/Pseudo-Lab/impatient-observatory/assets/35090655/604e5916-26d5-461c-86d4-94bf5277e8ce">  


- 해당 레포는 편집 권한이 없기 때문에, fork한 여러분의 repo, 즉, pseudolab/pseudolab.github.io가 아닌, 아이디/pseudolab.github.io 에서 편집 후 PR 해야합니다.

### 2) Edit & Commit

- 템플릿을 활용해 작성해주세요!
    - [[기획 템플릿](https://github.com/pseudolab/pseudolab.github.io/blob/master/_posts/2024-02-04-Template.md)](https://github.com/pseudolab/pseudolab.github.io/blob/master/_posts/2024-02-04-Template.md) / 중간 템플릿 / 마무리 템플릿 (중간/마무리 템플릿은 아직 준비중입니다.)
- 이미지 업로드가 필요한 경우
    - assets/images/post/8th-builder/아이디/이미지파일 형식으로 업로드 해주세요.
- 작성하신 파일을 `_posts`  폴더에 [[YYYY-MM-DD-제목.md](http://xn--yyyy-mm-dd--4308b263g.md/)](http://YYYY-MM-DD-제목.md) 형식으로 업로드 해주세요.
- 작성/편집은 jupyter notebook, vs code, bear 등 markdown을 편집할 수 있는 무엇을 사용하시든 자유입니다!

### 3) Pull Request

- pseudolab/pseudolab.github.io 페이지에서 Pull requests로 들어갑니다.
- `compare across forks` 를 클릭합니다.
<img width="1243" alt="image" src="https://github.com/Pseudo-Lab/impatient-observatory/assets/35090655/5277318a-2645-49b2-8846-1974c2598df4"> 

- 우측의 head repository에서 여러분의 `아이디/pseudolab.github.io`  를 클릭해주시고, branch를 선택후 (특별히 branch를 만들지 않았다면 master 혹은 main) `create pull request`  버튼을 눌러 PR을 진행합니다.
<img width="939" alt="image" src="https://github.com/Pseudo-Lab/impatient-observatory/assets/35090655/75c93aa1-b8f2-44e2-98dc-bb2465c0c4c0">

    

### 4) Review & Merge 기다리기

- 리뷰를 받고, 수정 사항이 있을 경우 수정을 진행합니다.
- Merge가 완료되면, 포스팅이 잘 수행되었는지 확인합니다.

# 온보딩 미션 : (https://github.com/Pseudo-Lab/impatient-observatory/discussions/11)


### 오픈 소스 기여 : 우리 모두가 공감할 수 있는 주제

- [ ]  메타정보
    - [ ]  좋은 소식을 모아둔 블로그들
    - [ ]  캐글, 공모전 등 활동 정보
    - [ ]  좋은 정보 소스: 단톡방, 게시판
- [ ]  ai의 활용 방법 : 페어 프로그래밍 잘하기 처럼
- [ ]  면접 질문 찾기
- [ ]  AI 회사 / 서비스 알아보기
- [ ]  (모임 구체적 주제:) llm 관련 내용들을 모아보자!

**Comment by HangryDev** [comment]
# 8/3 회의록

마법학교 - 어떻게 발전시켜볼까? 대화를 시작해보자. 사람들이 이야기를 내는 방향으로 디벨롭 될 것. 

유신: 온보딩이 애매한데, 애매한 지점이 정리가 안된다. 

경표:
1. 온보딩 보고 들어온거긴 하다. 
따라가는 사람도 노력을 해야 한다. 
2. 사람들을 많이 만나고 싶다!!!
3. 이런 걸 해볼 수 있다!!!  정보를 얻기 쉽지 않다 - 이미 하고 싶다 - 박람회 (정보 큐레이팅) 

유신: 
1. '공유 허브에 기여하기' 뜬금없다! 깃허브 부분을 배제하자. 청강하고 들어가는게 더 편하다. 숨이 턱 막힌다
2. 처음 왔을 때 정보 교류가 메인이라고 생각했다!! 온보딩을 메인으로 하려고 한다.  
3. 참가자들이 온보딩 뭐해야되나요?? 물어봤을 때 모르겠었다. 

한결: 궁극적인 목적이 뭐냐. 
온보딩의 궁극적 목적은. 
1. 자기만의 플젝 시작. (스터디라던가. 꼭 개발 아님! ) 
2. 그걸 위해서는 사람이 필요한데, 그 사람들과 연결해준다. 
3. 분위기를 알려주고, 그걸 볼수있게 시간을 내 준다. 

경표: 아이디어 - 정보를 올리는 곳이 깃헙 말고 디코여야 좋겠다. 

주영: 어떤 주제로 프로젝트를 진행할지 계속 던져주는 역할이 필요할 것 같아요. 이런저런 얘기를 하는 시간을 가져주는게 어떨지 싶어요. 

경표: 도메인을 AI에 한정하지 않았으면 좋겠다. 디자인 도메인 - 마케팅이라던가, 인스타 이런 이야기도 하자!!! 

홍규: 다른 포지션과 교류 - 매력적이다. 

한번 못온 사람들이 다시 왔을때 적응 못하는거<<< 문서만 보고 나설 수 있어야 하는데, 그게 좀 어렵다. 고 부분은 정리가 필요하다. 

경표: 모임 - 게임 기획같은것도 좋겠다.

**Comment by HangryDev** [comment]
# 8/10 회의록 

### 이번주 안건들

1. 온보딩 프로세스 누수- 들어오신 분들이 모임에 나오기까지, 그리고 나오고 활발하게 활동하는 과정에까지 이탈이 있습니다. (백프로 참여하게 만들 수는 물론 없지만) 설명이 불충분한 부분 등이 있을 겁니다. 참여자 입장을 물어보고, 프로세스 중 개선할 수 있는 부분을 찾아보겠습니다.
2. 우리는 뭐 한다를 정립할 필요 - 초조한 전망대는 3개의 목표(정보 공유, 온보딩, 네트워킹)가 있습니다. 다 챙기려다 보니 좀 혼란스러워요. 그래서 정리를 위해 어제 대화를 나눴고, 그 내용을 아래처럼 정리해봤어요. 앞으로도 대화하며 방향을 변경하며 발전시킬 겁니다.
    1. “정보 공유”: 서로 유익한 기사를 금요일까지 모아오고, 토요일날 온라인에서 1분 요점 요약 & 토론합니다.
    2. 온라인 모임 (”네트워킹 + 온보딩”): 오래 계셨던 분들이 새로운 멤버들에게 붙어서 얘기해줍니다. (저의 개인적 목표는 모든 참여자가 기존 스터디 연결 or 자기만의 모임이나 플젝을 시작하게 해 주는 겁니다. )
    3. 오프라인 (”네트워킹”): 직무별로 섞여서 노는 시간, 좀 더 친해지는 시간이 되어야겠습니다. 또 우리 하는거 보여주는 시간이기도 하고요.
    4. 결과물 내고 : 사람들이 보고 올 수 있어야 함. 외부 인정도 받으면 좋고. 선한 영향력 키우기.
    5. 이걸 기반으로 위에 추가적 활동들을 쌓아올리면서 재밌게 놀자.
3. 더 조직적 역할 정의 - 많은 분들이 이번 주 기사 올리시는걸 잊으신듯! 각자 뭘 해야 하는지 아직 습관이 안 된 상황으로 보입니다. 어떤 역할을 해 주어야 하는지 명확하게 정의해야 하겠습니다. 제가 리마인드 하겠습니다.
4. 비개발자면서 AI에 관심 있는 분들 챙기기 - API. 창구 역할도 놓치고 싶지는 않네요. 그 분들에게 닿는 것도 생각해보고 싶어요. 예를 들면 프롬프팅, 툴링 리뷰 등은 공통의 관심사가 될 수 있을 것 같아요. 월 1회 이런 세션을 가지는 것도 좋겠구요. 과하면 폐기해야겠지만요.
7. 오프라인 기획 마무리하겠습니다. 

## 안건 1: 오프라인 모임

- 회의 내용 요약 
    - "초조한 전망대" 행사 느낌이 아니었다 
    - "우리 활동"을 정리해 전시하면 어떨까
    - AI를 잘 쓰는 사람들을 직무 상관없이 모셔왔으면 좋겠다. 그분들의 페인포인트가 궁금하다
    - 주영님: 의료 테마, 유즈 케이스 공유하는 시간 발표할 수 있다! 
    - 일단 우리끼리 친해지는 시간이 더 필요하다. 
    - 너무 빡세지 않나? 네시간씩 
    - 해커톤이나 강의: 좋은 아이디어지만 꼭 메일 필요는 없다
- 남은 부분: 함께 만들어보자

#### Decide

메인 이벤트 : 
장소 : 
비용 : 

## 안건 2: 우리는 뭐 한다 정립!

### 비 개발자, AI/ML 외부 분들로 저변 넓히는 부분에 대해

- 기술 애호가란 전제 하에, 다양한 배경의 사람들이 왔으면 좋겠다! (도메인, 직무, 나이대, 성별, 지역, 국적?! 기타등등)
- 예시 : 더베러 / AI 코리아
- 그런 네트워킹을 위해서 우리의 취지(기술 이야기 나누기, 온보딩) 를 버리거나 완전히 새로운… 뭔가를 만들 압박을 받은 필요는 없다 (지난주 이야기 나온 개발자&도메인 전문가 페어링한 해커톤을 예시로 들어보자. 우리가 재밌기 위해, painpoint를 모으기 위해서 진행하는 건 좋다. 그리고 그분들도 배우는게 있기 위해서 올 수 있다. 그치만 너무 과도해서 부담일 수는 있겠다)
- 피드백: 좀 더 재밌는 분위기를 만들기 위해 할 수 있는 것.

### 결과물 업그레이드 - 뉴스레터 가자

- “밖에서 우리 뭐하는지 이해를 못한다” 문제 해결 
- 깃헙 페이지로 만들까? 
- 깃헙에 올리는거를 스티비 같은 플랫폼을 통해 배포도 해버리자.
    - 슬로건: “이런거 만들고요, 친해지고, 가짜연에서 활동하세여!”
- 후기 
    - 목적 - 우리 분위기 알게 해주는 것
    - 많이 읽어보셨는가? 결정에 어떤 영향을 미쳤나 

#### Decide

메일링 : 
플랫폼 : 깃헙 페이지? 

## 안건 3: 참여자들의 역할 재확인

- 참여자들이 참여하기 어려워한다! 
- 전제: 일을 나눠주는게 곧 온보딩이다 
    - 이걸 계기로 기존 멤버들과 새 멤버가 소통하는 자세가 포인트
- 뭘 찾아볼지 가이드를 줘야겠다. 
- 새로 오시는 분에게 일을 나눠주기!
1. 예시 주제 나눠주기
    - AI 분야 입문
    - 코딩 어시스턴트
    - 도메인별 AI 활용
    - 전반적인 AI 일어나는 일 
    - 기술 발전 - 제한
2. who to follow / read up page에서 하나 떼주기 
3. 기존 할일 나눠주기!!!  (https://github.com/Pseudo-Lab/impatient-observatory/discussions/17)

#### Decide

참가자 / 할 일 재정의
1. 호현 : AI Supremacy 정주행 
2. 

## 안건 4: 소개글

#### Decide: 
문서만 봤을때 이해 안 된 부분을 알려주세요 

## 안건 5: 지양했으면 하는 것들 
- 기술적으로 파고드는 주제 과편향 ❌  
    - 아예 없애자는 말은 아님! 40% 미만
- 15분~ 20분씩 발표하는거 ❌  
#### 대신 
- 기반 없는 사람도 이해할 수 있는 내용을 늘리자 ⭕ 
- 활용 방법에 집중 ⭕  
- 짧은 소개 이후, 긴 대화 ⭕ 


## 안건 6 : 할일 나누기 

### 일회성 - 내가/ 참여자가 할일 정리

1. 다음 오프라인 모임 기획
    1. 🏖️ 기존 회의내용 살려서 정리해오기  
    2. 🤲 기획을 마무리 - 멤버 분들에게 맡기자! 
2. 소개글
    1. 🏖️ 다시 쓰기 : 마법학교 OT 문서와 깃헙 디스커션의 소개글 합치기  
    2. 🏖️ 퇴고: 처음 오신 분들, 이 문서 이해 안되는 부분 있음. 소통하면서 글 고치기 
    3. 🏖️ 배포: repo.MD에 놓고, 깃헙 페이지로 배포
3. 결과물 뉴스레터 
    1. 가독성 높은 템플릿과 플랫폼 설정하자. (lilysAI / 스티비)  
    2. 🤲 후기 → 읽어보셨는지, 도움이 되었나 물어보기
4. 깃헙 구성: 후기, 회의록, 결과물 등등을 한 문서 / 아래 댓글 형태로 정리하기.  
5. 🤲 where to follow 문서: 분야별 정리하고, 담당자를 정해버리자
6. 슈도랩 onboarding missions : 애매한 부분 재작성 (깃헙 빼고)
7. **초조한 전망대로 참여하는 과정의 누수 탐지:** 들어오신 분들이 모임에 나오기, 나오고 활발하게 활동하기까지 있음. (백프로 참여하게 만들 수는 물론 없음.) 설명이 불충분해서 등등이 있을 것. 참여자 입장을 물어보고, 프로세스 중 개선할 수 있는 부분을 찾아보자.
    1. TODO: 톡방에 계신, 새로 오신 멤버분들한테 경험을 물어보자! 뭐를 필요로 하지?
    2. 안 오신 분들 누구한테 물어볼지 정하자 - 나눠서 이야기하자
8. 기존 운영업무
   - [x]  오늘 합니다! 글 올리기 🍉
   - [x] **discussion/(m월 n일) 모임에 "p회차 모임" 댓글** 만들기
   - [x]  ~~깃헙 디스커션 댓글에 멤버들이 찾아온 자료 정리~~ 🐨 -> 폐기
   - [x]  디스코드: **뉴스레터** 형태로 만들어서 #자유게시판 업로드 🐨 ->
        - ~~이번 주 멤버들이 얘기한 내용 - 클로바로 요약~~
        - 아래에 링크 남김! “온보딩이 필요하시다면 이 링크를 확인해주세요!”
   - [x] 모임 후기! 깃헙에만 업로드 🐸 -> 
   - [x]  공간 대여 & 준비 (ex 책상 밀어놓고 에어컨 트는거) 👯‍♂️ ->
   - [x]  (오프라인 있는 경우) 끝나고 놀 장소 섭외 👯‍♂️ -> 


#### Decide
하나씩 맡아주십셔

**Comment by HangryDev** [comment]
# 8/26 Done

- [x] 소개글 다시 쓰기 (한결)
- [x]  where to follow - 정리 (한결)
- [x]  onboarding missions (한결)
- [x]  다음 오프라인 모임 기획 (다같이)
- [x]  페이지 정리 (한결)
- [x] **discussion/(m월 n일) p회차 모임.md ** 만들기

**Comment by HangryDev** [comment]
# 2024 <그랜드 개더링> 참가준비 회의록!

## 메인 이벤트 : AI 이미지 & 음악 Jam Session + AI 도구 소개  

**설명**

1. 방 제일 잘 보이는 곳에 디스플레이를 두고, 진행자의 툴 시연 후 같이 이미지를 프롬프팅하는 시간을 갖습니다. 
2. 'AI 툴과 정보를 쓰면서 소개받는 세션'을 보여주면서 전망대 컨셉을 전달할 수 있습니다 (+ 손님 없을 때 우리끼리 유익한 시간)
3. 들어온 사람들과 네트워킹

**Why It Works**:

- It's visually appealing and draws attention.
- Visitors are actively contributing by suggesting prompts, making them feel involved.

### **세부사항**

**장소**

![KakaoTalk_20241111_065555566](https://github.com/user-attachments/assets/8041148f-78ca-4140-9458-b1fa8523503b)

**구역 분배** 

1. 가운데 메인 디스플레이: AI 이미지 생성 
    1. 다 같이 노트북 X 의견 모아서 프롬프트 하나씩 입력해보는 걸로
        1. 핸드폰으로 연결해서 써봐도 좋음. 앞에 디스플레이할 방법이 없을 뿐. 
    2. Display the results on a big screen, updating dynamically every few minutes.
2. 서브 PC : 음악(스피커) 연결한 PC + 먼가 간식 부스 두자  
    1. 신청곡 받는 사람 (=프롬프트 받는 사람) 
    2. 환호 한번 해주자
    3. 큐 1개 이하로 유지
3. 진행과 ‘툴 소개’ - 음악, 미술, 캐릭터 AI 하나씩 (선모님) 
    1. 3줄 소개, UI 알려주기, 시연 한번: 툴 당 5분 아래
    2. 약간은 덜 알려진걸로 
    3. 처음 시작하는거고, 다른 사람들이 받아서 새로운 도구를 소개하는걸 유도하는게 좋다. 
5. 벽에 전망대 활동 전시 : 보여줄 거 4개 포스터나 (슬라이드쇼) 모니터로 보여주기
    1. 우리가 찾은 데이터 소스
    2. 찾은 기사들
    3. 참가자 분들이 스터디로 잘 넘어갔는지 (온보딩 성과)
    4. 모임 후기!
    6. mini-gallery 디스플레이도 두자 - around the room with AI-created artwork from your community, sparking discussions.
5. 디스플레이 근처에서는 사생대회 참가

### 멤버들 - 당일날 할 일

✅ 메인 이벤트 진행 (디스플레이 1) 

✅ 음악 담당 + 쿠키 부스 (입구) 지키기

**AI 툴 사용에  참가** 

- 선모님 진행하는거 만들고 차분히 따라하면 됨

**손님맞이(Ambassadors)**

- 팀 멤버 2-3명 “floaters” to move around the room and engage with people not currently focused on the main event.
- 이 분들은 소개 멘트 + 손님맞이 루틴 숙달
    - 프롬프트 만들 때, 발표하기 수줍어하는 사람들한테 프롬프트 아이디어, 컨셉트, 테마 등을 물어봐주기 (참여 유도)
    - 전망대 컨셉트 등을 소개
    - 우리 디스커션 페이지의 모임 후기, 기사 등등 여러개 보여주기

### 준비 TODO

**사전 전시물 준비**

1. 마법학교 & 전망대 소개 
    - [ ]  포스터! 짧고, 시각적으로 만들기 (예신)    
    타게팅: 마법학교 수강생과 동일
    1. 가짜연구소 첨 알고, 탐색하시는 분들
    2. 개발 모르지만 AI 관심 많은 분들
    3. 알아냈는데 참여 포인트를 원하는 분들 
    4. awesome-list를 만들고 관리하는데 관심있을 분들 (정보수집탐색) 
2. ✅ 우리가 찾은 데이터 소스 
    1. ❓ 대표적인, 도움 되는 거 목록만 포스터로 뽑기. 
    2. 전체 목록도 치트시트 형태로 뽑는게 가능할 듯. 포스터에다가 QR을 달아서, “실제 링크는 디스커션에 있는 풀 목록을 보세요” 하고 링크 주자 
        ![DASDF](https://github.com/user-attachments/assets/42a678cc-795a-489f-bcc7-42f1885e3c7f)        
    3. ❓데이터 소스를 정리하기 **(다같이 정할 부분)** 
        1. 여러 가지 카테고리를 둬 봐도 가능할 것 같다. tag를 두는거지. 
        2. 지식 수준별로, 초중급으로 뎁스를 나눠두면 좋을것같다.  
3. 찾은 기사들
    1. 시각화: 
        - [ ]  캔바로 양식 정하자        
        ![123412341234](https://github.com/user-attachments/assets/b10bd9e1-ca82-42ec-8f78-89b2bba92bec)
        
        - [ ]  ~~뉴스레터 (스티비 양식)을 하나 정하기 - 시각화 용 템플릿~~
        - [ ]  ~~이참에 사실 오토메이션 해버리고 싶긴 하다. 깃헙에 올린거 매주 업로드되게.~~
    2. 매체 : 대표적으로 두장 정도만 포스터로 하자 
        - [ ]  올 타임 베스트 기사 뽑기
        - [ ]  최신 뉴스 뽑기 & 설명 추가: 설명이 없는 부분은 정리해서 써넣기.
        - [ ]  이 포스터 아래에도 QR 해서, 우리가 찾는 기사들 링크로 이어지게.
4. 참가자들의 이야기 
    1. ✅ 멤버들의 참여 이유! 이래서 들어왔고, 이래서 도움이 되었다 (윤석) 
        - [ ]  구글 폼으로 정보 받고, 누락된거 물어보기
        
        (자주 안오신 분들도 한번 물어보기! VOC) 
        
        - 필요한 정보
            1. 왜 왔나 
            2. 와서 느낀게 뭔가 (참가후기) 
            3. 지금 스터디 뭐 하고있는가! 
            4. 아쉬운 점, 뭐가 있으면 더 좋았겠다. 
    2. 참가자 분들이 스터디로 잘 넘어갔는지 (온보딩 성과) 온 사람, 스터디로 넘어간 사람
    3. 모임 후기! << 사진이 있으므로, 매체와 배치만 정하면 됨. 
    - [ ]  캔바로 인포그래픽 만들기 - 윤하님께 한번 여쭤보고

**디자인 초이스…! (윤하)** 

- [ ]  캔바 인포그래픽, 뉴스레터 템플릿 고르기
- [ ]  cheatsheet builder - 포스터 템플릿 고르기!

신경쓸거! 

- 편집 필요가 제일 적어야 함
- 포스터 사이즈 뭘로 하는게 좋을까여? - 레이아웃 잡아주기
- QR 넣는거만 신경쓰기!

**QR generator 찾기**

- 적당한 SaaS 하나 찾아야 함! ‘몇번 이상 보면’ 유료인거 확인해야 함

## 멤버별 할 일! 
- 선모 - 메인 진행
- 윤석 - 참가자들의 이야기! 수집
- 호현 - 
- 윤하 - 디자인
- 예신 - 소개 포스터! 짧고, 시각적으로 만들기
- 유신 - 슈도콘 스태프진과의 우리 사이 소통 담당
- 재희 - 데이터 소스 치트시트 만들기
- 경표 - 올타임 베스트 기사 & 최신 뉴스 포스터로 뽑기



### 이후 할일

- [ ]  인쇄 맡기기

**Reply by HangryDev** [reply]
인쇄 수요일에 맡겨야 함
화요일까지 모든 포스터가 준비되어야 함 
디자인은 오늘 정해져야 함.

**Reply by yunha-p** [reply]
1. '준비 TODO 3.1'의 이미지 내에서 템플릿을 골라야 하나요, 아니면 제가 따로 찾아봐야 할까요?
-> 사실 제가 피그마로 만들어도 되긴 합니다

2. 포스터 부착 위치는 결정되었나요? 부착 위치에 따라 사이즈 견적 내야할 것 같습니다.

**Reply by HangryDev** [reply]
@yunha-p 저기 안에서 고를 필요는 없어요! 부착 위치는 저 위의 사진에서 파란 의자 위의 벽입니다

**Reply by HangryDev** [reply]
제가 아는게 canva라서 써놓은건데 텍스트 편집하기만 쉬우면 뭐든 상관없어여! 모두의 수고가 최소화되는 선에서 제일 이쁜거 가져
